app/src/main/java/com/example/kotlincalculator/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.kotlincalculator.mvp.model
+
+import com.example.kotlincalculator.utils.EMPTY_STRING
+
+class CalculatorModel {
```

Comment:
here you can use a data class
https://www.baeldung.com/kotlin-data-classes

Reply:
done